common/views/components/AudioPlayer/AudioPlayer.tsx

```diff
@@ -1,14 +1,14 @@
-// @flow
 import Router from 'next/router';
 import { trackEvent } from '@weco/common/utils/ga';
-import { useEffect, useState } from 'react';
+import { useEffect, useState, ReactElement, FunctionComponent } from 'react';
 import useInterval from '@weco/common/hooks/useInterval';
-import { type IIIFMediaElement } from '@weco/common/model/iiif';
-type Props = {|
-  audio: IIIFMediaElement,
-|};
-
-const AudioPlayer = ({ audio }: Props) => {
+import { IIIFMediaElement } from '@weco/common/model/iiif';
```

Comment:
bikeshed - I don't think we should be including from `@weco/common` when we are already in common.

I know it works, but it only works as it's a dep of the content/catalogue app, and is thus installed into the root `node_modules`.

Reply:
makes sense ! changing now